src/os/layer/tile.js

```diff
@@ -326,6 +362,56 @@ os.layer.Tile.prototype.setColor = function(value, opt_options) {
   }
 };
 
+/**
+ * Adjust the layer brightness.  A value of -1 will render the layer completely
+ * black.  A value of 0 will leave the brightness unchanged.  A value of 1 will
+ * render the layer completely white.  Other values are linear multipliers on
+ * the effect (values are clamped between -1 and 1).
+ *
+ * @param {number} value The brightness of the layer.
```

Comment:
The description at the top is good, but it would be better to also have a short version on the `value` parameter itself.

Also, consider having the function check that the value is in the proper range via `goog.assert` would be nice. Even though the underlying filter function clamps the values properly, a check like that will alert developers that they are sending something incorrect (like `[0, 100]` from a slider or spinner).

Ditto for the rest of the setters in this file.